src/mon/MonCommands.h

```diff
@@ -869,6 +869,31 @@ COMMAND("osd pool get-quota " \
         "name=pool,type=CephPoolname ",
         "obtain object or byte limits for pool",
         "osd", "r", "cli,rest")
+COMMAND("osd pool application enable " \
```

Comment:
Oh, sorry. I mean that in addition to this set of generics that take whatever string the admin gives us, we can *also* have

osd pool application enable-rbd

etc. or maybe that just seems to simply repetitive? But it would mean we can tweak the internal representations a bit more (like adding more metadata when we enable pools) and existing deployment tools would still function; it would protect against typoes; it would just generally seem a little Moreno polished.